torch/distributions/distribution.py

```diff
@@ -189,5 +205,9 @@ def _validate_log_prob_arg(self, value):
                 raise ValueError('Value is not broadcastable with batch_shape+event_shape: {} vs {}.'.
                                  format(actual_shape, expected_shape))
 
+        if self._validate_args:
```

Comment:
Do we need this check? Are we planning to ever call this method without having `_validate_args` evaluate to true?